pycbc/filter/resample.py

```diff
@@ -66,16 +64,14 @@ def lfilter(coefficients, timeseries):
         cfreq = zeros(flen, dtype=ftype)
         tfreq = zeros(flen, dtype=ftype)
 
-        fft(Array(cseries), cfreq)
-        fft(Array(timeseries), tfreq)
```

Comment:
Is there a potential danger here of a user giving the "don't preserve FFT option" and this not working?

Reply:
Hmm, yes. That's true. I'll see if there is as alternative path here.